src/lib/registration/widgets/registration_code.rb

```diff
@@ -70,8 +70,7 @@ def register
         end
 
         if !SwMgmt.find_base_product
-          log.info("Not base product found, skipping registration.")
-          return true
+          return error(_("No base product found, skipping registration."))
```

Comment:
You can reuse the [Registration::Helpers.report_no_base_product](https://github.com/yast/yast-registration/blob/327ab34c020a89f8b7e3f4bff55deea82e457237/src/lib/registration/helpers.rb#L232) method here to avoid new translations.